static/js/components/NewsFeed.jsx

```diff
@@ -16,11 +19,90 @@ const defaultPrefs = {
   numItems: "",
 };
 
+const newsFeedItem = (item) => {
```

Comment:
```suggestion
const NewsFeedItem = (item) => {
```
b/c this is a React component

Reply:
This will also need a propTypes statement